.github/workflows/publish-ghcr.yml

```diff
@@ -0,0 +1,26 @@
+name: Publish Docker image to GHCR
```

Comment:
Is there a reason for separating this from the `publish.yml` file? 🤔 I'd have thought it would be quite nice to have them named `Publish / npm` and `Publish / Docker` or something.

Reply:
Nice catch, I will merge these workflows too under a same one.